ExtensionCommands.ts

```diff
@@ -26,6 +26,7 @@ export class ExtensionCommands {
     static readonly ADD_ENVIRONMENT: string = 'environmentExplorer.addEnvironmentEntry';
     static readonly DELETE_ENVIRONMENT: string = 'environmentExplorer.deleteEnvironmentEntry';
     static readonly ASSOCIATE_IDENTITY_NODE: string = 'environmentExplorer.associateIdentityNodeEntry';
+    static readonly REPLACE_ASSOCIATED_IDENTITY: string = 'environmentExplorer.replaceAssociatedIdentityEntry';
```

Comment:
During environment setup, I can call `replace identity` from the command palette.
I'm then presented with a list of nodes (in my case, they all had no identity associated).
It then asked to select a node to associate an identity with. This should probably ask what node I want to replace the identity for.

Reply:
When I have associated an identity with a node during setup, the node isn't visible.
When I call the `replace identity` from the command palette, I can't see what identity is associated with each nodes.